### PR TITLE
Regulations page

### PR DESCRIPTION
**Pull request by allisonking**
Closes https://ethyca.atlassian.net/browse/PROD-1494

### Description Of Changes

https://www.loom.com/share/5a3b8c49537749f59da1ecaa97a16da9?sid=486f6b01-9aea-4eb6-a0f3-16893698f2ed

### Code Changes

* [ ] _list your code changes here_

### Steps to Confirm

* [ ] _list any manual steps for reviewers to confirm the changes_

### Pre-Merge Checklist

* [ ] All CI Pipelines Succeeded
* Documentation:
  * [ ] documentation complete, [PR opened in fidesdocs](https://github.com/ethyca/fidesdocs/pulls)
  * [ ] documentation [issue created in fidesdocs](https://github.com/ethyca/fidesdocs/issues/new/choose)
* [ ] Issue Requirements are Met
* [ ] Relevant Follow-Up Issues Created
* [ ] Update `CHANGELOG.md`
* [ ] For API changes, the [Postman collection](https://github.com/ethyca/fides/blob/main/docs/fides/docs/development/postman/Fides.postman_collection.json) has been updated

### CODE REVIEW

**Comment by simonkeane33** [comment]
Looking great @allisonking , just a couple of minor enough points : 

1. For the sub group titles, it looks like we're using the same font size and weight as the main "continent" title. The group title should be smaller so that there's some typographic hierarchy from continent to group to locations. So, all that said, can we make sure we're using 18px for the continent title and 16px for the sub group titles, weight should be semibold.
2. I've added a recommendation for the modal copy into the design if you have a chance to update that.

Otherwise all good. Thank you.

**Comment by allisonking** [comment]
Thanks @simonkeane33 , I've made those changes! They're actually on [the branch](https://github.com/ethyca/fides/pull/4531) this one is branched off of, but I've merged it into this branch as well.

image


image

File: clients/admin-ui/src/features/system/SystemFormTabs.tsx [hunk 1]
```diff
@@ -37,45 +38,20 @@ const ToastMessage = ({
...
-    // allow lines to wrap
-    display: "initial",
-    cursor: "pointer",
-  };
-  return (
-    <Box>
-      <Text fontWeight="700">System has been saved successfully</Text>
-      <Text textColor="gray.700" whiteSpace="inherit">
-        Your system has been added to your data map. You can{" "}
-        <Button
```
Comment (allisonking) [inline clients/admin-ui/src/features/system/SystemFormTabs.tsx:55]: this is technically unrelated to this PR, but this PR also needed a button link in a toast, which we did here. so I extracted what we did here out into its own component `ToastLink` and used it here, too.

**Review by Kelsey-Ethyca — APPROVED** [review]
🦖

**Pull request merged**